test/functional/mempool_updatefromblock.py

```diff
@@ -109,10 +109,10 @@ def transaction_graph_test(self, size, n_tx_to_mine=None, start_input_txid='', e
         self.log.info('Checking descendants/ancestors properties of all of the in-mempool transactions...')
         for k, tx in enumerate(tx_id):
             self.log.debug('Check transaction #{}.'.format(k))
-            assert_equal(self.nodes[0].getrawmempool(True)[tx]['descendantcount'], size - k)
-            assert_equal(self.nodes[0].getrawmempool(True)[tx]['descendantsize'], sum(tx_size[k:size]))
-            assert_equal(self.nodes[0].getrawmempool(True)[tx]['ancestorcount'], k + 1)
-            assert_equal(self.nodes[0].getrawmempool(True)[tx]['ancestorsize'], sum(tx_size[0:(k + 1)]))
+            assert_equal(self.nodes[0].getmempoolentry(tx)['descendantcount'], size - k)
+            assert_equal(self.nodes[0].getmempoolentry(tx)['descendantsize'], sum(tx_size[k:size]))
+            assert_equal(self.nodes[0].getmempoolentry(tx)['ancestorcount'], k + 1)
+            assert_equal(self.nodes[0].getmempoolentry(tx)['ancestorsize'], sum(tx_size[0:(k + 1)]))
```

Comment:
If you're going to do this to save time, you might as well just call the RPC once.
```suggestion
            entry = self.nodes[0].getmempoolentry(tx)
            assert_equal(entry['descendantcount'], size - k)
            assert_equal(entry['descendantsize'], sum(tx_size[k:size]))
            assert_equal(entry['ancestorcount'], k + 1)
            assert_equal(entry['ancestorsize'], sum(tx_size[0:(k + 1)]))
```

Reply:
This puts me at 13sec (vs 33sec on master).